lib/cadet_web/plug/check_admin.ex

```diff
@@ -0,0 +1,21 @@
+defmodule CadetWeb.Plug.CheckAdmin do
```

Comment:
Sorry, can you rename this to `EnsureAdmin`? this is to be consistent with Guardian's `EnsureAuthenticated`